### PR TITLE
Use %q for quoted strings where appropriate

### PR DESCRIPTION
**Pull request by swayne275**
Not a huge value add, but I figured I might as well clean up some of our string formatting.

I didn't touch anywhere that looked like formatting mattered. lmk what you think!

### CODE REVIEW

File: api/client_test.go [hunk 1]
```diff
@@ -254,7 +254,7 @@ func TestDefaulRetryPolicy(t *testing.T) {
 				t.Fatalf("expected to retry request: '%t', but actual result was: '%t'", test.expect, retry)
 			}
 			if err != test.expectErr {
-				t.Fatalf("expected error from retry policy: '%s', but actual result was: '%s'", err, test.expectErr)
+				t.Fatalf("expected error from retry policy: %q, but actual result was: %q", err, test.expectErr)
```
Comment (calvn) [inline api/client_test.go:257]: There's a slight difference between these two in that any quoted strings within those values will be escaped if using `%q` but not with `'%s'`, but if the spirit of the output here is to quote the error message itself and not so much about escaping quoted values within `err` then I think this is fine to do.

https://go.dev/play/p/xZQeqklVxRq

Comment (swayne275) [reply api/client_test.go:257]: oh that's great to know! I'd imagine most/all of these cases were from not knowing/thinking about using the `%q` verb, but if it looks like there was some other intention it shouldn't be changed

**Review by ccapurso — APPROVED** [review]
Looks good to me! The point about escaping is an interesting one. I think you're likely correct in that the use of `'%s'` was likely not intentional to prevent escaping though.

**Comment by ccapurso** [comment]
> So what do we think about this, worth updating with current main and merging?

Seems reasonable to me

**Comment by swayne275** [comment]
assuming tests pass i'm planning to merge this today or tomorrow unless there are any objections? @ccapurso @calvn

**Pull request merged**